tests/queries/0_stateless/03172_error_log_table_not_empty.sh

```diff
@@ -4,17 +4,19 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 # shellcheck source=../shell_config.sh
 . "$CURDIR"/../shell_config.sh
 
+# system.error_log is created lazy, flush logs query makes it sure that the table is created.
+$CLICKHOUSE_CLIENT -q "SYSTEM FLUSH LOGS;"
+
 # Get the previous number of errors for 111, 222 and 333
 errors_111=$($CLICKHOUSE_CLIENT -q "SELECT sum(value) FROM system.error_log WHERE code = 111")
 errors_222=$($CLICKHOUSE_CLIENT -q "SELECT sum(value) FROM system.error_log WHERE code = 222")
 errors_333=$($CLICKHOUSE_CLIENT -q "SELECT sum(value) FROM system.error_log WHERE code = 333")
 
-# Throw three random errors: 111, 222 and 333 and wait for more than collect_interval_milliseconds to ensure system.error_log is flushed
+# Throw three random errors: 111, 222 and 333 and call flush logs to ensure system.error_log is flushed
 $CLICKHOUSE_CLIENT -mn -q "
 SELECT throwIf(true, 'error_log', toInt16(111)) SETTINGS allow_custom_error_code_in_throwif=1; -- { serverError 111 }
 SELECT throwIf(true, 'error_log', toInt16(222)) SETTINGS allow_custom_error_code_in_throwif=1; -- { serverError 222 }
 SELECT throwIf(true, 'error_log', toInt16(333)) SETTINGS allow_custom_error_code_in_throwif=1; -- { serverError 333 }
-SELECT sleep(2) format NULL;
```

Comment:
We need this. Without it, no errors are collected and tests fail close to 100% of the time:

```bash
+++ /home/ubuntu/ClickHouse/ClickHouse/tests/queries/0_stateless/03172_error_log_table_not_empty.stdout 2024-07-04 16:29:37.956423656 +0000
@@ -1,6 +1,6 @@
-1
-1
-1
-1
-1
-1
+0
+0
+0
+0
+0
+0
```